_static/torchvision_finetuning_instance_segmentation.ipynb

```diff
@@ -348,8 +348,8 @@
             "creating build/temp.linux-x86_64-3.6/content/vision/torchvision/csrc\n",
             "creating build/temp.linux-x86_64-3.6/content/vision/torchvision/csrc/cpu\n",
             "creating build/temp.linux-x86_64-3.6/content/vision/torchvision/csrc/cuda\n",
-            "x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DWITH_CUDA -I/content/vision/torchvision/csrc -I/usr/local/lib/python3.6/dist-packages/torch/include -I/usr/local/lib/python3.6/dist-packages/torch/include/torch/csrc/api/include -I/usr/local/lib/python3.6/dist-packages/torch/include/TH -I/usr/local/lib/python3.6/dist-packages/torch/include/THC -I/usr/local/cuda/include -I/usr/include/python3.6m -c /content/vision/torchvision/csrc/vision.cpp -o build/temp.linux-x86_64-3.6/content/vision/torchvision/csrc/vision.o -O0 -DTORCH_API_INCLUDE_EXTENSION_H -DTORCH_EXTENSION_NAME=_C -D_GLIBCXX_USE_CXX11_ABI=0 -std=c++11\n",
-            "x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DWITH_CUDA -I/content/vision/torchvision/csrc -I/usr/local/lib/python3.6/dist-packages/torch/include -I/usr/local/lib/python3.6/dist-packages/torch/include/torch/csrc/api/include -I/usr/local/lib/python3.6/dist-packages/torch/include/TH -I/usr/local/lib/python3.6/dist-packages/torch/include/THC -I/usr/local/cuda/include -I/usr/include/python3.6m -c /content/vision/torchvision/csrc/cpu/ROIAlign_cpu.cpp -o build/temp.linux-x86_64-3.6/content/vision/torchvision/csrc/cpu/ROIAlign_cpu.o -O0 -DTORCH_API_INCLUDE_EXTENSION_H -DTORCH_EXTENSION_NAME=_C -D_GLIBCXX_USE_CXX11_ABI=0 -std=c++11\n",
+            "x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DWITH_CUDA -I/content/vision/torchvision/csrc -I/usr/local/lib/python3.6/dist-packages/torch/include -I/usr/local/lib/python3.6/dist-packages/torch/include/torch/csrc/api/include -I/usr/local/lib/python3.6/dist-packages/torch/include/TH -I/usr/local/lib/python3.6/dist-packages/torch/include/THC -I/usr/local/cuda/include -I/usr/include/python3.6m -c /content/vision/torchvision/csrc/vision.cpp -o build/temp.linux-x86_64-3.6/content/vision/torchvision/csrc/vision.o -O0 -DTORCH_API_INCLUDE_EXTENSION_H -DTORCH_EXTENSION_NAME=_C -D_GLIBCXX_USE_CXX11_ABI=0 -std=c++14\n",
```

Comment:
D_GLIBCXX_USE_CXX11_ABI looks like it's still referencing CXX11. Is the 14 equivalent just `D_GLIBCXX_USE_CXX14_ABI` ?

Reply:
same in a few places below as well

Reply:
No, there was an ABI change in gcc between c++03 and c++11 and this flag allows you to select which ABI you want. There was no such change and there is no such flag between c++11 and c++14